data/filedefs/filetypes.Swift.conf

```diff
@@ -0,0 +1,45 @@
+# For complete documentation of this file, please see Geany's main documentation
+[styling=C]
+
+[keywords]
+primary=associativity break case catch class continue convenience default deinit didSet do else enum extension fallthrough false final for func get guard if import in infix init inout internal lazy let mutating nil operator override postfix precedence prefix private public repeat required return self set static struct subscript super switch throws true try var weak where while willSet
+secondary=Array Bool Dictionary ErrorType Int Float Double Set String Tuple UnicodeScalar abs max min print
+# documentation keywords for javadoc
+doccomment=author deprecated exception param return see serial serialData serialField since throws todo version
+
+[settings]
+lexer_filetype=C
+
+# default extension used when saving files
+extension=swift
+
+# MIME type
+mime_type=text/x-swift
+
+# the following characters are these which a "word" can contains, see documentation
+#wordchars=_abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789
+
+# single comments, like # in this file
+comment_single=//
+# multiline comments
+comment_open=/*
+comment_close=*/
+
+comment_use_indent=true
+
+[build-menu]
+FT_00_LB=chmod +x
+FT_00_CM=chmod +x "%f"
+FT_00_WD=
```

Comment:
On Unix-like systems, Swift files can be executed as scripts using shebangs, as shown in the code below. That requires the execute bit to be set.

```
#!/usr/bin/env swift

import Glibc
import Foundation

print("hello, world")
```